android/src/main/kotlin/edu/ucsd/WifiConnection/WifiConnectionPlugin.kt

```diff
@@ -120,7 +120,7 @@ class WifiConnectionPlugin: FlutterPlugin, MethodCallHandler, ActivityAware, Plu
         currentActivity = null
     }
 
-    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>?, grantResults: IntArray?): Boolean {
+    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>, grantResults: IntArray): Boolean {
```

Comment:
Hi spiritcoding23, thank you for submitting this PR.

We are tracking a similar issue in our campus-mobile repository [#1785](https://github.com/ucsd/campus-mobile/issues/1785)

Can you provide more information about this change? I believe these checks were added by a Flutter null-safety migration tool.